### PR TITLE
Issue #5065: Context validation error on Configure Layout page and WSOD on cancel

### LINKED ISSUE #5065: Context validation error on Configure Layout page and empty page output on cancel
A couple of errors can be encountered when adding/configuring layouts. 

1. Go to Add layout. Pick any name and template.
2. Set the path to `node/%`. Context appears and is set to type Node.
3. Change the path to `foobar/%`. Context switches to "String pass-through".
4. Select context type of Node again from the type selector.
5. Click "Create Layout" to save the layout.
6. Click "Configure Layout" tab to re-configure it. (Note that the context is still set to "Node".)
7. Click "Add visibility condition"
8. Select visibility condition of "Node: Type".
8. Select a node type, e.g., "Page".
9. Click "Add visibility condition".
10. Click "Save Layout".
11.  This produces an error `The condition 'Type is <em class="placeholder">Page</em>' does not have the required contexts at this path. Remove the condition to save the layout.`.
12. Hit the cancel button, and get WSOD at path /system/ajax. Watchdog error is `Notice: Undefined index: #ajax in ajax_form_callback() (line 393 of <snip>/core/includes/ajax.inc).`

The extra HTML in the form error is pretty clearly because the condition summary is getting double-sanitized, so that should be an easy fix.

Don't know why the WSOD is happening, but guessing that it might be related to the question of whether one should be able to put a Node context on a bare % placeholder (whose value could well be an invalid nid) and how rigorously that is enforced.

### PR DESCRIPTION
**Pull request by jayelless**
Fixes https://github.com/backdrop/backdrop-issues/issues/5065

### CODE REVIEW

File: core/modules/layout/plugins/access/layout_access.inc [hunk 1]
```diff
@@ -41,10 +41,10 @@ class LayoutAccess extends LayoutHandler {
    * @param $contexts
    *   An array containing available contexts.
    * @return string
-   *   The summary string for this access check.
+   *   The sanitized HTML summary string for this access check.
    */
   function summary() {
-    return '';
+    return t('');
```
Comment (bugfolder) [inline core/modules/layout/plugins/access/layout_access.inc]: You don't need to translate empty strings. So `return '';` was correct.

Comment (jayelless) [reply core/modules/layout/plugins/access/layout_access.inc]: Strange however.  Removing the t() call results in one of the date-theme tests failing, but restoring the t() call allows all tests to pass???

**Review by klonos — APPROVED** [review]
LGTM 👍🏼

**Comment by quicksketch** [comment]
Merged into 1.x and 1.20.x. Thanks @jayelless!

**Pull request merged**